docs/core/tools/global-tools.md

```diff
@@ -1,135 +1,270 @@
+The output shows the command used to invoke the tool and the version installed, similar to the following example:
 
 ```output
 You can invoke the tool using the following command: dotnetsay
-Tool 'dotnetsay' (version '2.0.0') was successfully installed.
+Tool 'dotnetsay' (version '2.1.4') was successfully installed.
 ```
 
-Global Tools can be installed in the default directory or in a specific location. The default directories are:
+The default location for a tool's binaries depends on the operating system:
 
 | OS          | Path                          |
 |-------------|-------------------------------|
 | Linux/macOS | `$HOME/.dotnet/tools`         |
 | Windows     | `%USERPROFILE%\.dotnet\tools` |
 
-These locations are added to the user's path when the SDK is first run, so Global Tools installed there can be called directly.
+This location is added to the user's path when the SDK is first run, so global tools can be invoked from any directory without specifying the tool location.
+
+Tool access is user-specific, not machine global. A global tool is only available to the user that installed the tool.
+
+### Install a global tool in a custom location
+
+To install a tool as a global tool in a custom location, use the `--tool-path` option of [dotnet tool install](dotnet-tool-install.md), as shown in the following examples.
+
+On Windows:
+
+```dotnetcli
+dotnet tool install dotnetsay --tool-path c:\dotnet-tools
+```
+
+On Linux or macOS:
+
+```dotnetcli
+dotnet tool install dotnetsay --tool-path ~/bin
+```
+
+The .NET Core SDK doesn't add this location automatically to the PATH environment variable. To invoke a tool-path tool, you have to make sure the command is available by using one of the following methods:
+
+* Add the installation directory to the PATH environment variable.
+* Specify the full path to the tool when you invoke it.
+* Invoke the tool from within the installation directory.
+
+## Install a local tool
+
+**Applies to .NET Core 3.0 SDK and later.**
+
+To install a tool for local access only (for the current directory and subdirectories), it has to be added to a tool manifest file. To create a tool manifest file, run the `dotnet new tool-manifest` command:
+
+```dotnetcli
+dotnet new tool-manifest
+```
+
+This command creates a manifest file named *dotnet-tools.json* under the *.config* directory. To add a local tool to the manifest file, use the [dotnet tool install](dotnet-tool-install.md) command and **omit** the `--global` and `--tool-path` options, as shown in the following example:
+
+```dotnetcli
+dotnet tool install dotnetsay
+```
+
+The command output shows which manifest file the newly installed tool is in, as shown in the following example:
+
+```
+You can invoke the tool from this directory using the following command:
+dotnet tool run dotnetsay
+Tool 'dotnetsay' (version '2.1.4') was successfully installed.
+Entry is added to the manifest file /Users/name/tools/sub/dotnet-tools.json
```

Comment:
The output here demonstrates linux/mac pathing, but the example below for the `dotnet tool list` command uses windows pathing.